packages/bc-wallet-openapi/docs/diagram.md

```diff
@@ -15,6 +19,7 @@ classDiagram
         +hidden : Boolean
         +createdAt : DateTime
         +updatedAt : DateTime
+        +tenantId : String
```

Comment:
How can we avoid it to crash the showcase atm?
- first merge the auth or something? 
- make it optional until we have all the auth depencies inplace?

Reply:
You can create a default tenant for now with ID 00000000-0000-0000-0000-000000000000 using the tenant controller (when not exists)
When we have a means to fetch the real tenant id we can use that